server/client_manager.py

```diff
@@ -377,6 +379,32 @@ def gimp_message(self, message):
                        '((status?))']
             return random.choice(message)
 
+        def check_version(self, req_release, req_major, req_minor, exact):
+            if len(self.version) < 3:
+                return False
+            if req_release:
```

Comment:
is there a scenario where req_release etc. aren't defined?

Reply:
I can add a sanity check for it ,but I can't foresee a situation where you'd want to check for only a minor/major and have that hold up long-term.